etc/docker/kernel-image-puller/kernel_image_puller.py

```diff
@@ -162,6 +167,7 @@ def puller():
     logger.info("KIP_NUM_RETRIES: {}".format(num_retries))
     logger.info("KIP_PULL_POLICY: {}".format(policy))
     logger.info("KIP_LOG_LEVEL: {}\n".format(log_level))
+    logger.info("KIP_AUTH_TOKEN: {}\n".format(auth_token))
```

Comment:
Sorry, but we should probably go ahead and remove **_both_** of the additional newlines since this now introduces a blank line amongst the env info messages (and isn't friendly for future additions):
```
[I 2021-05-03 23:21:19,460 kernel_image_puller.MainThread] KIP_PULL_POLICY: IfNotPresent
[I 2021-05-03 23:21:19,460 kernel_image_puller.MainThread] KIP_LOG_LEVEL: INFO

[I 2021-05-03 23:21:19,460 kernel_image_puller.MainThread] KIP_AUTH_TOKEN: foobar

[I 2021-05-03 23:21:24,466 kernel_image_puller.MainThread] Fetching kernelspecs from 'http://enterprise-gateway.enterprise-gateway:8888/api/kernelspecs' ...
```
Everything looks good.